src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -200,6 +200,12 @@ public void should_successfully_return_all_envelopes_with_processed_status_for_a
             );
 
         blobProcessorTask.processBlobs();
+
+
+        List<Envelope> envelopes = envelopeRepository.findEnvelopesToUpload(10);
+        assertThat(envelopes).hasSize(1);
+        envelopes = envelopeRepository.findAll();
+        assertThat(envelopes).hasSize(1);
```

Comment:
although description says add logs but this bit is added here :)

Reply:
test is failing just checking. this Pr was not about this test.I will remove